examples/common/utilities.adb

```diff
@@ -11,6 +11,13 @@ with GL.Types.Colors;
 
 package body Utilities is
 
+   procedure Clear_All (Colour : GL.Types.Colors.Color) is
+   begin
+        GL.Buffers.Clear ((True, True, True, True));
+        GL.Buffers.Set_Color_Clear_Value (Colour);
```

Comment:
I still really don't understand how these work.
I did these in the same order as the Clear Buffer Flag and Colour routines which I thought I had ordered in agreement with a previous recommendation of yours.
As I understand, Clear clears four bits of a "bitfield"; it implements glClear.
Set_Color_Clear_Value implements glClearColor.
I'll try and do some more research to understand this better. That is, see if the OpenGL documentation on glClear and glClearColor clarifies my understanding.